.hars/response_was_not_application-json.har

```diff
@@ -1,384 +0,0 @@
-{
```

Comment:
I removed this since I'm not sure if we want what I assume is a HAR of an error we saw once, in the repo. Additionally I think I've solved the mystery - if it was a mystery! I got the same error when submitting to the registration tree but not filling in the KBA answer. It seems that the tree fails in that scenario.